src/flint/fmpz.jl

```diff
@@ -1249,10 +1248,17 @@ function rising_factorial(x::fmpz, y::Int)
     y < 0 && throw(DomainError("Argument must be non-negative: $y"))
     z = fmpz()
     ccall((:fmpz_rfac_ui, :libflint), Nothing,
-          (Ref{fmpz}, Ref{fmpz}, UInt), z, x, y)
+          (Ref{fmpz}, Ref{fmpz}, UInt), z, x, UInt(y))
```

Comment:
This change doesn't hurt, but `ccall` automatically inserts such conversions.

Reply:
I was not aware of that.

Reply:
Actually, I am sure Keno complained at me that the type of the argument didn't match the type specified, so I always put these in now.